core/samples/data/ser/json/main.cpp

```diff
@@ -0,0 +1,40 @@
+#include <cubos/core/data/ser/json.hpp>
+#include <cubos/core/log.hpp>
+#include <cubos/core/memory/stream.hpp>
```

Comment:
```suggestion
```

Reply:
Stream is used to show the output: line 34: "Stream::stdOut.print(json.dump());"